integration_tests/mockApis/curiousApi.ts

```diff
@@ -33,10 +33,18 @@ const stubLearnerProfile = (prisonNumber = 'G6115VJ'): SuperAgentRequest =>
           ],
           languageStatus: null,
           plannedHours: null,
-          rapidAssessmentDate: null,
-          inDepthAssessmentDate: null,
-          primaryLDDAndHealthProblem: null,
-          additionalLDDAndHealthProblems: [],
+          prisonId: 'MDI',
```

Comment:
I'm confused by this change
This stub file is the stub of the raw data we get from the Curious API .... and the Curious API hasnt changed, and it certainly does not have fields `prisonName`, `atLeastOnePrisonHasSupportNeeds`, and `hasSupportNeeds`

I can see why you've set values for the Curious API fields `rapidAssessmentDate`, `primaryLDDAndHealthProblem` and `additionalLDDAndHealthProblems`, but not the ones mentioned above 🤔 

And also, I _think_ I'd leave `inDepthAssessmentDate` as `null` rather than `undefined`. I know in respect of how we handle the fields/data downstream of this it makes no difference (null and undefined are both falsey),  but given this stub is to mirror the data we would expect to get from a given API I think it's important to use the same data types/values that we would expect to get from the real API. And I think the real API will return a field with the value `null` rather than `undefined` (or not returning the field at all)